### ISSUE TITLE
Diving Goggles

### ISSUE THREAD

**Issue by StevenLiekens** [SamHurne/gw2pao#38]
A completion assistant for diving goggles would be nice to have.
https://wiki.guildwars2.com/wiki/Dive_Master

**Comment by SamHurne** [comment]
Agreed. Once upon a time I was working on compiling diving goggles locations for the Tasks Tracker, but never really finished it up...

There are several "find stuff" achievements now. It would probably be easiest to just create some files people can import into the Tasks Tracker with all of the locations.
